lib/loader.js

```diff
@@ -72,11 +79,17 @@ module.exports = function loader(content) {
       regExp: config.symbolRegExp
     });
   }
-  svgCompiler.addSymbol({ id, content, path: resourcePath + resourceQuery })
+  svgCompiler.addSymbol({
+    id,
+    content: isSVG ? content : contentBuffer,
+    path: resourcePath + resourceQuery
+  })
     .then((symbol) => {
       const runtime = runtimeGenerator({ symbol, config, context: loaderContext.context, loaderContext });
       done(null, runtime);
     }).catch(done);
 };
 
 module.exports.NAMESPACE = NAMESPACE;
+
+module.exports.raw = true;
```

Comment:
What is purpose of this change?

Reply:
> What is purpose of this change?

this will make the first argument of loader become a `Buffer` from a `string`, so that we can detect whether it is an valid image file using `image-size`. (`image-size` is also required by `svg-baker` so I chose it for detecting image)